test/simulator/timestamp/simulator_interface/simulator_interface.cpp

```diff
@@ -158,21 +104,82 @@ interface_rollback_transaction()
 }
 
 void
-interface_set_timestamp()
+interface_timestamp_transaction()
 {
 }
 
 void
-interface_timestamp_transaction()
+interface_session_query_timestamp()
 {
 }
 
 void
-interface_timestamp_transaction_uint()
+print_rules()
 {
 }
 
-void
-print_rules()
+int
+main(int argc, char *argv[])
 {
+    bool exit = false;
+
+    do {
+        std::cout << std::endl;
+
+        std::cout << "1: Session Management " << std::endl;
```

Comment:
You can make a vector of choices and then a loop that would print the index + 1 and the content of the vector.

Reply:
Updated in [d3b07ca](https://github.com/wiredtiger/wiredtiger/pull/8417/commits/d3b07cad2974c45eb19a02631df03808cff9758d)